src/charm.py

```diff
@@ -866,7 +881,8 @@ def _restart_traefik(self):
                 _TRAEFIK_SERVICE_NAME: {
                     "override": "replace",
                     "summary": "Traefik",
-                    "command": BIN_PATH,
+                    # trick to drop the logs to a file but also keep them available in the pod logs
+                    "command": '/bin/sh -c "{} | tee {}"'.format(BIN_PATH, self._log_path),
```

Comment:
+1 This is great

Reply:
I don't remember who I ripped it off of; it's not flour from my own bag (*italian idiom)